app/models/tweet.rb

```diff
@@ -0,0 +1,7 @@
+class Tweet < ApplicationRecord
+  validates :text, {presence: true, length:{maximum:140}}
+
+  def user
+    return User.find_by(id: self.user_id)
+  end
```

Comment:
アソシエーションを使ったほうがよいですね。

```suggestion
  belongs_to :user
```